src/style.css

```diff
@@ -1,3 +1,95 @@
-body {
+* {
+  padding: 0;
   margin: 0;
+  list-style: none;
+  box-sizing: border-box;
```

Comment:
Avoid using the universal selector (*) for setting padding, margin, list-style, and box-sizing as it impacts performance. Set styles only for elements that require them.